src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -733,6 +792,41 @@ public void setNrAudioChannels(int numberOfChannels) {
 	}
 
 	/**
+	 * Sets the number of audio channels.
+	 *
+	 * @param numberOfChannels the number of channels to set, use 0 if unknown.
```

Comment:
@Sami32 ```0``` is for unknown - it means that the bitrate is actually unknown or not set. It's not useful to store the default with every media, it's better to be able to check if it's actually unknown and then either use the default or not use it at all depending on the situation in which the bitrate is needed. When you store the default, there's no way to tell if the bitrate is actually unknown or just happens to be the same as the default.